### DISCUSSION TITLE
i want to download brew on windows enabled wsl but ruby wont install

### DISCUSSION THREAD

**Discussion by smts3x** [Homebrew/discussions#3790] [Getting started]
i already did all the suggested tips like making it first in my path folder and such but nothing worked
![image](https://user-images.githubusercontent.com/115655590/195900602-33f4cf84-8a36-4ec5-b6df-cb0b776ff5f8.png)

**Comment by Bo98** [comment]
WSL1 has known issues that cause various executables to not work. Try switch to WSL2 if you can.

If you must use WSL1, try run `sudo apt update && sudo apt upgrade`. It seems there was a fix put out recently for `gzip`: https://bugs.launchpad.net/ubuntu/+source/gzip/+bug/1966849, but other executables may also be affected so you might still have some issues. I think `node` can be another one that can cause issues.

**Reply by smts3x** [reply]
thank you so much, solved the problem
